rust/src/nasl/builtin/cryptographic/aes_gcm.rs

```diff
@@ -2,12 +2,8 @@
 //
 // SPDX-License-Identifier: GPL-2.0-or-later WITH x11vnc-openssl-exception
 
-// FunctionErrorKind::GeneralError
-use crate::nasl::syntax::NaslValue;
-use crate::{
-    function_set,
-    nasl::utils::{Context, FunctionErrorKind, Register},
-};
+// NaslError::GeneralError
+use crate::nasl::prelude::*;
```

Comment:
personal opinion: while combating boilerplate code this does make lookups more annoying.